phpseclib/Crypt/Common/SymmetricKey.php

```diff
@@ -204,6 +204,13 @@ abstract class SymmetricKey
         self::ENGINE_OPENSSL_GCM => 'OpenSSL (GCM)'
     ];
 
+    /** @var string|null */
+    public $fixed;
+    /** @var string|null */
+    public $invocation_counter;
+    /** @var string|null */
+    public $name;
```

Comment:
It's adding these because they're used in SSH2.php. SSH2.php _shouldn't_ be using dynamic properties as they're being [deprecated in PHP 8.2](https://wiki.php.net/rfc/deprecate_dynamic_properties) so it's a good find but I think it'd be better to define them in SSH2.php. I'll try to look into this later today or tomorrow.